optaplanner-core/pom.xml

```diff
@@ -32,6 +32,10 @@
       <groupId>org.kie</groupId>
       <artifactId>kie-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kie.kogito</groupId>
+      <artifactId>kogito-api</artifactId>
+    </dependency>
```

Comment:
This is non-optional?

Reply:
`KieRuntimeBuilder` moved from Drools to `kogito-api`, and is used in the file that compiles DRL; I can move the default logic back to ScoreDirectorFactoryFactory and make that file only be used when Quarkus involved.